openstudiocore/CMake/FindEnergyPlus.cmake

```diff
@@ -42,8 +43,9 @@
 # (To distributed this file outside of CMake, substitute the full
 #  License text for the above reference.)
 
-file(GLOB ENERGYPLUS_POSSIBLE_PATHS "${CMAKE_BINARY_DIR}/../../../[eE]nergy[pP]lus?*")
-
+# TODO: JM 2019-02-26 to @kbenne
+# not sure why it's looking three folders above CMAKE_BINARY_DIR (=the build folder).
+file(GLOB ENERGYPLUS_POSSIBLE_PATHS "${CMAKE_BINARY_DIR}/[eE]nergy[pP]lus-*" "${CMAKE_BINARY_DIR}/../../../[eE]nergy[pP]lus?*")
```

Comment:
@kbenne I'm fine leaving this as is, just want to know if it makes sense in which case I'll remove the TODO

Reply:
That does not make sense to me

Reply:
It doesn't make sense to me either, so I guess we should remove it.

Really this part isn't going to be used anyways in the vast majority of cases (unless something goes wrong)

What this does is to glob possible folders in which E+ might be located, but then it inserts in first position the `ENV{ENERGYPLUSDIR}` which is set when you downloaded and extracting the zip, so that's the one it'll find, and it should go just fine in the foor loop so it'll hit the break statement and will never try the other paths.